diff -ru NVIDIA-Linux-x86-1.0-6629-pkg1.orig/usr/src/nv/conftest.sh NVIDIA-Linux-x86-1.0-6629-pkg1/usr/src/nv/conftest.sh
--- NVIDIA-Linux-x86-1.0-6629-pkg1.orig/usr/src/nv/conftest.sh	2004-11-18 10:34:39.000000000 +0100
+++ NVIDIA-Linux-x86-1.0-6629-pkg1/usr/src/nv/conftest.sh	2004-11-18 10:49:21.385817576 +0100
@@ -17,7 +17,7 @@
 
 if [ "$OUTPUT" != "$SOURCES" ]; then
     CFLAGS="$CFLAGS -I$OUTPUT/include2 -I$OUTPUT/include \
--I$HEADERS -I$HEADERS/asm/mach-default"
+-I$HEADERS -I$OUTPUT/include/asm/mach-default"
 else
     CFLAGS="$CFLAGS -I$HEADERS -I$HEADERS/asm/mach-default"
 fi
