--- NVIDIA-Linux-x86-71.86.06-pkg1/usr/src/nv/nv.c	2008-07-02 21:26:01.000000000 +0200
+++ NVIDIA-Linux-x86-71.86.06-pkg1/usr/src/nv/nv.c	2008-07-22 15:33:10.000000000 +0200
@@ -1774,7 +1774,7 @@
     .open   = nv_kern_vma_open,
     .close  = nv_kern_vma_release,  /* "close" */
 #if !defined(NV_VM_INSERT_PAGE_PRESENT)
-    .nopage = nv_kern_vma_nopage,
+    .nopage = nv_kern_vma_nopage
 #endif
 };
 
