--- NVIDIA-Linux-x86-71.86.01-pkg1/usr/share/applications/nvidia-settings.desktop~	2007-10-20 00:35:17.000000000 +0200
+++ NVIDIA-Linux-x86-71.86.01-pkg1/usr/share/applications/nvidia-settings.desktop	2007-10-20 00:39:52.000000000 +0200
@@ -1,8 +1,9 @@
 [Desktop Entry]
 Type=Application
-Encoding=UTF-8
 Name=NVIDIA X Server Settings
 Comment=Configure NVIDIA X Server Settings
-Exec=__UTILS_PATH__/nvidia-settings
-Icon=__PIXMAP_PATH__/nvidia-settings.png
-Categories=Application;System;
+Exec=nvidia-settings
+Icon=nvidia-settings
+Terminal=false
+Categories=Settings;DesktopSettings;
+# vim:encoding=utf-8
